### PR TITLE
A few minor improvements

### PR DESCRIPTION
**Pull request by josharian**
See individual commit messages for details.

### CODE REVIEW

**Review by glvr182 — APPROVED** [review]
LGTM, thank you for this PR.

File: README.md [hunk 1]
```diff
@@ -129,5 +129,6 @@ func quit(g *gocui.Gui, v *gocui.View) error {
 * [fac](https://github.com/mkchoi212/fac): git merge conflict resolver
 * [jsonui](https://github.com/gulyasm/jsonui): Interactive JSON explorer for your terminal.
 * [cointop](https://github.com/miguelmota/cointop): Interactive terminal based UI application for tracking cryptocurrencies.
+* [lazygit](https://github.com/jesseduffield/lazygit): simple terminal UI for git commands.
```
Comment (mjarkk) [inline README.md:132]: We could also add lazydocker here (https://github.com/jesseduffield/lazydocker)

**Review by mjarkk — COMMENTED** [review]
Looks good, we could also add Lazydocker to the list of project using this library.

**Comment by mjarkk** [comment]
Well it seems like i need to do this myself then :)

**Pull request merged**